returnn/torch/data/pipeline.py

```diff
@@ -114,22 +114,23 @@ def __iter__(self):
             num_chunks = None
 
             for data_key in chunking_data_keys:
-                chunk_size = self._chunk_size[data_key]
-                chunk_step = self._chunk_step[data_key]
-
-                data = data_dict[data_key]
-                chunks = [
-                    data[start_index : start_index + chunk_size] for start_index in range(0, len(data), chunk_step)
-                ]
-
-                if num_chunks is None:
-                    num_chunks = len(chunks)
-                else:
-                    assert num_chunks == len(
-                        chunks
-                    ), "Chunking resulted in different number of chunks for different data keys."
-
-                data_chunks[data_key] = chunks
+                if data_key == "data":
```

Comment:
You never should hardcode any data key name here (except maybe of the special keys like `seq_tag`, but `data` is really not special). Also, there should be no logic specific for some data key (except maybe the special keys like `seq_tag`).

Reply:
I see you added this as a workaround for chunking, which breaks because of `seq_tag`. This does not belong here into the PR. Can you undo this change?

For testing now without having this fixed, just don't use chunking. Let us fix chunking in a separate commit or PR.